Main.py

```diff
@@ -151,7 +152,7 @@ def write_rom(game: Game, romWriter: Optional[RomWriter] = None) -> str:
 
 
     rom_clean_path = "roms/Ascent.sfc"
-    rom_name = f"Ascent{game.options.seed}.sfc"
+    rom_name = f"Ascent{game.options.get_file_hash()}.sfc"
```

Comment:
I wanted to add more than just the seed to the filename. See `GameOptions.get_file_hash` for more details.